corehq/apps/userreports/models.py

```diff
@@ -645,15 +645,15 @@ def all(cls, use_server_filter=True):
     def by_domain(cls, domain):
         """
         Returns a list of DataSourceConfiguration objects,
-        NOT StaticDataSourceConfigurations.
+        NOT a list of StaticDataSourceConfiguration objects.
```

Comment:
I think I understand what this is trying to say, but it comes off oddly because (English language) it sounds like it will not return a list of only StaticDataSourceConfiguration objects, but by implication, possibly a list of at least one DataSourceConfiguration object and maybe one ore more StaticDataSourceConfiguration objects?? Saying it that way just sounds ridiculous, and you should feel free to stop reading right now. However, since I got this far, what do you think of changing it to:

Returns a list of DataSourceConfiguration objects excluding any StaticDataSourceConfiguration objects.

Or maybe I still have it wrong? If so, that suggests it could be reworded some other way for clarity.

Reply:
I chose to remove this comment as I think it's more confusing to have it.

The relationship here is that `StaticDataSourceConfiguration` is a superset of `DataSourceConfiguration` and calling this removes the extra fields (those extra fields being things like what domains to add the "static" config to). So in short, this method provides the same behavior as `DataSourceConfiguration`, but for custom reports defined in code